### PR TITLE
Quiz logic

### PR DESCRIPTION
**Pull request by shadowas-py**

### CODE REVIEW

File: src/components/mainPhoto/getRandomImage.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import IMAGES_IDs from './all_Images';
+
+function getRandomImage(category = '', imagesCollection = []) {
+  console.log(category, imagesCollection);
```
Comment (sebcio-o) [inline src/components/mainPhoto/getRandomImage.js:4]: Delete console.log ;D

File: src/components/mainPhoto/mainPhoto.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  min-width: 200px;
+  width: 29vw;
+  max-width: 360px;
+  grid-area: image;
+  justify-self: center;
+  align-self: start;
+  margin-left: 2em;
+  margin-top: 2rem;
+  margin-right: 1em;
+  margin-bottom: 0;
```
Comment (sebcio-o) [inline src/components/mainPhoto/mainPhoto.css:11]: You can use margin: 2rem 1rem 0 2rem;

**Review by FilipHalon — APPROVED** [review]
There is one more thing that could be implemented - a quiz could end after a player answered a last available question.

**Comment by FilipHalon** [comment]
Actually 2 more things:
1) the game over modal does not seem to appear when in the characters mode;
2) it is possible to click an answer again after selecting an answer for the current question which is taken for an answer for a next question.

You could also add `user-select: none` to the buttons to prevent button text being selected if the cursor is dragged instead of just clicked.

**Pull request merged**